Utils/Piece.py

```diff
@@ -1,10 +1,10 @@
 class Piece:
```

Comment:
For sure there should be some thought out code structure. It's not properly structured, when everything is in module called "Utils". There should be multiple submodules. For example I could see something like: `chess-engine` (which contains submodules like `board`, `pieces` etc.), `utilities` (but maybe it should rather be a singular file), `